src/app/games/connect-six/tests/ConnectSixMinimax.spec.ts

```diff
@@ -0,0 +1,169 @@
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+        ];
+        const state: ConnectSixState = new ConnectSixState(board, 2);
+        const node: ConnectSixNode = new ConnectSixNode(state);
+
+        // When asking what is the best move
+        const bestMove: ConnectSixMove = node.findBestMove(1, minimax);
+
+        // Then it should be that victory
+        expect(bestMove).toEqual(ConnectSixDrops.from(new Coord(4, 0), new Coord(5, 0)).get());
+    });
+    it('should block double-open fives at level two', () => {
+        // Given a minimax at level two
+        // And an board where current opponent could win if nothing is done (..XXXXX..)
+        const board: Table<PlayerOrNone> = [
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, O, O, O, O, O, _, _, _, _, _, _, _, _, _, _, _, _],
+        ];
+        const state: ConnectSixState = new ConnectSixState(board, 3);
+        const node: ConnectSixNode = new ConnectSixNode(state);
+
+        // When asking what is the best move
+        const bestMove: ConnectSixMove = node.findBestMove(2, minimax);
+
+        // Then the minimax level two should block
+        expect(bestMove).toEqual(ConnectSixDrops.from(new Coord(1, 18), new Coord(7, 18)).get());
+    });
+    it('should count all possible moves including only neighbooring-coord', () => {
+        // Given a board with 60 possibles combinaisons of two coords
+        // With the first being neighbors of the original set
+        // and the second able to also be a neighboor of the first coord
```

Comment:
point comprendre ce given

Reply:
Bah, tu peux placer deux fois une pièce, à chaque fois strictement voisine à une pièce déjà sur le plateau, bah y a 60 façon de placer ces deux pièces.

Comment j'exprime ça autrement que "60 combinaisons possible de deux coord"

Reply:
combinaisons -> combinations
les 60 combinaisons ça va, mais "the original set", c'est quoi ? Et "able" me semble pas approprié dans la dernière ligne : the second should also be… (et neighbour, pas neighboor)

Reply:
awhé c'est pas fou la formulation là! enhanced!

Reply:
of piece -> of a piece
also able to neighbor -> also be neighbors? Une coordonnée n'a pas la capacité à « être capable de » il me semble !

Reply:
Rah, met pas les coms sur du code morts, c'est tout chiant de le retrouver après :D

Reply:
mais c'est fait :p
Y a deux trois neighboooors que tu as ignoré !